src/appengine/handlers/cron/ml_train.py

```diff
@@ -28,17 +28,25 @@ class Handler(base_handler.Handler):
   def get(self):
     """Handle a GET request."""
     for job in data_types.Job.query():
+
       if not utils.string_is_true(
           job.get_environment().get('USE_CORPUS_FOR_ML')):
         continue
 
+      if utils.string_is_true(job.get_environment().get('USE_GRADIENTFUZZ')):
```

Comment:
It still looks like this wouldn't work properly for the case where both variables were set. Could you replace task_name with a task_names list above, and add one task of each?

Reply:
Ah I see -- one job, but multiple tasks. Got it!